Sources/Apodini/Components/Path/RelationshipNameModifier.swift

```diff
@@ -0,0 +1,36 @@
+//
+// Created by Andi on 05.01.21.
+//
+
+struct RelationshipNameContextKey: ContextKey {
+    // Move to Optional type once fixed https://github.com/Apodini/Apodini/issues/75
+    static var defaultValue: String = ""
```

Comment:
It actually doesn't trigger one 🙃 Maybe because its also used to define the associated value of the `ContextKey`
But have nonetheless fixed it f02f2cf